demo/ts/components/victory-brush-line-demo.tsx

```diff
@@ -102,15 +102,13 @@ class App extends React.Component<any, BrushLineDemoState> {
           point && Math.max(...filters[name]) >= point.y && Math.min(...filters[name]) <= point.y
         );
       }, true);
-
-      return this.state.datasets
-        .map((dataset: DataSet) => {
-          return isActive(dataset, filters) && dataset ? dataset.name : null;
-        })
-        .filter(Boolean);
     };
 
-    return [];
```

Comment:
I was looking at the odd failure that was persisting in CI. `nps compile-ts` is failing, and I think it was caused by removing this return.

Reply:
oops, you already caught it